_includes/header.html

```diff
@@ -61,15 +62,16 @@
 						</hgroup>
 
 						<div class="download">
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.zip" class="latest-zip">
-								<img border="0" width="90" alt="Download latest release zipfile" src="{{ site.tgmpa.zipimage }}">
-							</a>
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.tar.gz" class="latest-tar">
-								<img border="0" width="90" alt="Download latest release tarball" src="{{ site.tgmpa.tarimage }}">
-							</a>
+							<p class="download-icon">
+								<a href="{{ '/download/' | prepend: site.tgmpa.url }}" title="Download the latest release">
```

Comment:
Unless screen readers have an override for reading out title attributes on input elements with pattern attributes, even when their settings say not to read out title attributes, I'd rather see this pattern described within the label, or even placeholder text (i.e. "1ABC" for the spec example), or some other visual description. http://www.deque.com/blog/accessible-client-side-form-validation-html5/ has a few suggestions.

cc @afercia @rrwd